templates/default/pages/_app.js

```diff
@@ -29,12 +29,7 @@ function MyApp({ Component, pageProps }) {
 
   return (
     <ThemeProvider attribute="class" defaultTheme="dark">
-      <DefaultSeo
-        titleTemplate={'%s | ' + siteConfig.title}
```

Comment:
I think there is now an issue as i don't see that we are getting right title e.g. if i go to https://deploy-preview-123--spectacular-dragon-c1015c.netlify.app/docs/roadmap i just get "Roadmap" in the title not "Roadmap | Flowershow"

Reply:
Adding a templateTitle will do this for the homepage or any other page that doesn't have a frontmatter title because the title is always defined in `config.js` nextSeo property.

<img width="289" alt="Screen Shot 2022-08-24 at 3 44 24 PM" src="https://user-images.githubusercontent.com/42637597/186419912-225837b2-e616-4105-9dbf-b5dcc4ca2cbe.png">